### PR TITLE
Add unit tests

### PR DESCRIPTION
**Pull request by Rkareko**

### CODE REVIEW

**Comment by keymane** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- opensrp-app/src/test/java/org/smartregister/commonregistry/CommonObjectFilterOptionTest.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/OpenSRP/opensrp-client-core/pullRequest?prid=5471629&bid=17931986)

**Comment by keymane** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- opensrp-app/src/test/java/org/smartregister/commonregistry/CommonObjectFilterOptionTest.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/OpenSRP/opensrp-client-core/pullRequest?prid=5471629&bid=17931986)

**Comment by keymane** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- opensrp-app/src/test/java/org/smartregister/commonregistry/CommonObjectFilterOptionTest.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/OpenSRP/opensrp-client-core/pullRequest?prid=5471629&bid=17931986)

**Comment by keymane** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- opensrp-app/src/test/java/org/smartregister/commonregistry/CommonObjectFilterOptionTest.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/OpenSRP/opensrp-client-core/pullRequest?prid=5471629&bid=17931986)

File: opensrp-app/src/test/java/org/smartregister/service/HTTPAgentTest.java [hunk 1]
```diff
@@ -103,6 +104,7 @@ public void testUrlCanBeAccessWithGivenCredentialsGivenWrongUrl(){
     }
 
     @Test
+    @Ignore
```
Comment (githengi) [inline opensrp-app/src/test/java/org/smartregister/service/HTTPAgentTest.java:107]: Why is this being ignored

Comment (Rkareko) [reply opensrp-app/src/test/java/org/smartregister/service/HTTPAgentTest.java:107]: An issue has been created for fixing that test here https://github.com/OpenSRP/opensrp-client-core/issues/511

**Comment by keymane** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- opensrp-app/src/test/java/org/smartregister/commonregistry/CommonObjectFilterOptionTest.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/OpenSRP/opensrp-client-core/pullRequest?prid=5471629&bid=17931986)

**Comment by keymane** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- opensrp-app/src/test/java/org/smartregister/commonregistry/CommonObjectFilterOptionTest.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/OpenSRP/opensrp-client-core/pullRequest?prid=5471629&bid=17931986)

**Pull request merged**